src/poliastro/core/iod.py

```diff
@@ -293,6 +330,97 @@ def _compute_y(x, ll):
     return np.sqrt(1 - ll ** 2 * (1 - x ** 2))
 
 
+@jit
+def _check_bounds(psi, psi_new, psi_up, psi_low, tof_new, tof):
+    """Updates psi_new, psi_up, psi_low
+
+    "Check if newton guess for psi is outside bounds (too steep a slope)
+     and update values accordingly"
+
+    """
+    if np.abs(psi_new) > psi_up or psi_new < psi_low:
+        # lower_bound
+        steep_condition_1 = tof_new < tof and psi > psi_low
+        psi_low = psi_low + (psi - psi_low) * steep_condition_1
+
+        # upper_bound
+        steep_condition_2 = tof_new > tof and psi < psi_up
+        psi_up = psi_up + (psi - psi_up) * steep_condition_2
+
+        psi_new = (psi_up + psi_low) / 2
+
+        return psi_new, psi_low, psi_up
+
+
+@jit
+def _c2dot(psi, c2, c3):
+    return 0.5 / psi * (1.0 - psi * c3(psi) - 2.0 * c2(psi))
+
+
+@jit
+def _c3dot(psi, c2, c3):
+    return 0.5 / psi * (c2(psi) - 3.0 * c3(psi))
+
+
+@jit
+def _c2dot_para(psi):
+    return (
+        -1.0 / np.math.factorial(4) + 2.0 * psi / np.math.factorial(6) - 3.0 * psi
```

Comment:
`Unknown attribute 'factorial' of type Module(<module 'math' `

https://dev.azure.com/poliastro/poliastro/_build/results?buildId=259&view=logs&j=77e370ac-c6ce-58ed-b689-4eaf2ed67de9&t=ffe2475a-255d-544e-4141-d84b3eb0a61b&l=1457

You must use `math.gamma`, see https://github.com/poliastro/poliastro/pull/31/files#diff-bfecda9ed598400464e2ce964edbb901 and https://github.com/numba/numba/pull/987#issuecomment-84980245.